pkg/kfapp/kustomize/kustomize.go

```diff
@@ -859,7 +859,10 @@ func MergeKustomizations(kfDef *kfconfig.KfConfig, compDir string, overlayParams
 				}
 			}
 		} else {
-			log.Warnf("No overlay %v for component at %v, skipping...", overlayParam, compDir)
+			return nil, &kfapisv3.KfError{
+				Code:    int(kfapisv3.INTERNAL_ERROR),
+				Message: fmt.Sprintf("no overlay %v for component at %v Error %v", overlayParam, compDir, err),
```

Comment:
Maybe capitalize...? 
```suggestion
				Message: fmt.Sprintf("No overlay %v for Component at %v Error %v", overlayParam, compDir, err),
```

Reply:
Not sure about whether there is a convention or not. But I was following the one fmt.Sprintf right above (line 858).

Reply:
Ok - even there "N" is capital in "no"

Reply:
What I mentioned is this one:
https://github.com/kubeflow/kfctl/blob/master/pkg/kfapp/kustomize/kustomize.go#L858

Reply:
hmmm - @jlewi is there a convention here?